wodles/aws/tools.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+def check_role_session_duration(duration: int or None) -> bool:
```

Comment:
There is no need for a new `tools.py` module just for this function as this check can be performed in a `arg_valid_iam_role_duration` function in the `aws_s3.py` module